pysmt/cmd/installers/base.py

```diff
@@ -307,3 +308,70 @@ def find_python_config(self):
             if command is not None:
                 break
         return command
+
+
+def package_install_site(name='', user=False, plat_specific=False):
```

Comment:
`plat_specific` -> `platform_specific`

Reply:
`plat_specific` is standard name in [`distutils`](https://docs.python.org/2/distutils/apiref.html).